### PR TITLE
Add Monitor plus Banner to homepage

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary
Adds Monitor plus banners to homepage - midpage and pencil. This should be live February 6th

## Issue / Bugzilla link
#14115 

## Testing
http://localhost:8000/

### CODE REVIEW

**Comment by nathan-barrett** [comment]
This switch PR will need to be pushed on Feb 6th

File: bedrock/mozorg/templates/mozorg/home/home-new.html [hunk 1]
```diff
@@ -43,8 +46,10 @@
 {% endblock %}
 
 {% block page_banner %}
-  {%- if (LANG == 'de' or LANG.startswith('en-')) and show_pocket_banner -%}
-    {% include 'includes/banners/pocket-pencil-banner.html' %}
+  {# {%- if (LANG == 'de' or LANG.startswith('en-')) and show_pocket_banner -%}
```
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/home/home-new.html:49]: Guessing this comment is an artifact from local testing? I think this needs to be uncommented and the following rule changed to `elif`.

File: bedrock/base/templates/includes/banners/pocket-pencil-banner.html [hunk 2]
```diff
@@ -12,9 +12,9 @@
   {% set cta_text = 'See the winners' %}
 {% endif %}
 
-<aside class="pocket-banner c-banner hide-from-legacy-ie" id="pocket-banner">
+<aside class="monitor-plus-banner c-banner hide-from-legacy-ie" id="monitor-banner">
```
Comment (craigcook) [inline bedrock/base/templates/includes/banners/pocket-pencil-banner.html:15]: I don't think these name changes should have been committed? Seems like the Monitor banner started life as a copy of the Pocket banner, but the original Pocket banner should be unchanged. These name changes break the Pocket banner styling.

**Review by craigcook — CHANGES_REQUESTED** [review]

**Review by craigcook — APPROVED** [review]

**Review by reemhamz — APPROVED** [review]
r+ 🍻

**Pull request merged**